1-development/ListPostsByRating/src/Api/Models/Post.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Api.Models
+{
+    public class Post
+    {
+        public int Id { get; set; }
+        public int UpVotes { get; set; }
+        public int DownVotes { get; set; }
+        internal double Ratio
+        {
+            get
+            {
+                const int noVotes = 0;
+                return UpVotes > noVotes || DownVotes > noVotes
+                    ? (double)UpVotes / (UpVotes + DownVotes)
```

Comment:
How can we distinguish a Post that has 6 upvotes and 4 downvotes from a Post that has 600 upvotes and 400 downvotes?

Because:
`6 / (6 + 4) = 6 / 10 = 0.6`
and
`600 / (600 + 400) = 600 / 1000 = 0.6`

Both return a ratio of `0.6`.

Reply:
By using the more distinct property `DiffVotes` which gives us the difference between up votes and down votes.
Ratio alone is not enough to be a distinctive property.